notebooks/start-a-business-checker/users_visiting_two_start_a_business_content_items.sql

```diff
@@ -0,0 +1,13 @@
+
+select count(distinct Sessionid) as Sessions, CountofPages
+from (
+       select sessionid, count(distinct pagepath) as CountOfPages
+       from (
+                SELECT CONCAT(fullVisitorId, "-", CAST(visitId AS STRING)) AS sessionId,
+                       pagepath
+                FROM `govuk-bigquery-analytics.FlattenedDailyData.flattened_daily_ga_data_*`
+                where type = 'PAGE' and pagepath in UNNEST(@pages) and _TABLE_SUFFIX BETWEEN @date_from AND @date_to
+            )
+       group by sessionid
+   )
+group by countofpages
```

Comment:
The SQL looks correct to me. Some suggestions to consider:

- add one line of docstring at the top outlining what this query does
- be consistent with the use of upper or lower case characters for the SQL commands (I prefer upper case)

- Here is an alternative implementation that uses CTEs rather than nested queries. I  think it is much easier to follow (it also happens to be slightly faster):
```
WITH target_pages_sessions AS (
    SELECT 
        CONCAT(fullVisitorId, "-", CAST(visitId AS STRING)) AS sessionId,
        hits.page.pagePath,
    	FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`, UNNEST(hits) AS hits
        WHERE hits.type = 'PAGE' AND hits.page.pagePath in UNNEST(@pages) AND _TABLE_SUFFIX BETWEEN @date_from AND @date_to
            
),

count_distinct_pages_per_session AS (
    SELECT 
        sessionid, 
        COUNT(distinct pagepath) as countOfDistinctPages
        FROM target_pages_sessions
    GROUP BY sessionId

)

SELECT 
    COUNT(Sessionid) AS numberOfSessions, 
    countOfDistinctPages
FROM count_distinct_pages_per_session
GROUP BY countOfDistinctPages
ORDER BY countOfDistinctPages
```

- You see I have also suggested querying the original `ga_sessions_*` data. It turns out that for this query, is is cheaper. ~906MB vs your original 1.2GB :)